spec/redirect.test.sh

```diff
@@ -597,7 +597,7 @@ status=0
 
 #### can't mention big file descriptor
 echo hi 9>&1
-# 23 is the max descriptor fo rmksh
+# 23 is the max descriptor for mksh
 #echo hi 24>&1
 echo hi 99>&1
 echo hi 100>&1
```

Comment:
Doesn't make sense to me. If `echo hi 24>&1` is commented out because 23 is the max descriptor for mksh, then why does the next lines go up to 99 and 100?

Reply:
This is a bit of mksh trivia that's not important to test, since the tests agree on a common behavior for all shells

It's OK to just fix the typo here, thanks